lib/quotaManager.js

```diff
@@ -0,0 +1,57 @@
+const logger = require('@elastic.io/component-logger')();
```

Comment:
is it possible to use `this.logger` for correct execution logs?

Reply:
It's only way to add logger to static class instance